### PR TITLE
Display grade and work per week on course reviews #80 #88

### PR DESCRIPTION
**Pull request by ZiqiXiong**
screenshot 2016-09-29 11 47 12

### CODE REVIEW

File: aspc/courses/forms.py [hunk 1]
```diff
@@ -71,6 +71,10 @@ def label_from_instance(self, obj):
     (1.5, '1.5'),
     (2.0, '2.0'), (3.0, '3.0'), (4.0, '4.0'), (6.0, '6.0'))
 
+POSSIBLE_GRADES = (
```
Comment (KentShikama) [inline aspc/courses/forms.py]: A+, P, NC, Other?

Comment (ZiqiXiong) [inline aspc/courses/forms.py]: Done

**Comment by KentShikama** [comment]
On a second thought do we really want to allow people to enter grades? This does give ASPC course reviews a change of dynamic that I don't know if we should just blindly implement, especially since we aren't the original creators. I'm also slightly concerned that when we get hacked, we now have information that some people might consider confidential. If you don't think these are large enough issues, just go ahead and merge; technical side looks good to me.

File: aspc/courses/templatetags/stars.py [hunk 2]
```diff
@@ -15,3 +15,9 @@ def star(rating):
         num_star_half * ('<img height=15 src="'+STATIC_URL+'images/star-half.png">') + \
         num_star_off * ('<img height=15 src="'+STATIC_URL+'images/star-empty.png">')
     return mark_safe(html)
+
+@register.filter
+def grade(rating):
+    grades = dict([(1, u'A'), (2, u'A-'), (3, u'B+'), (4, u'B'), (5, u'B-'), (6, u'C+'), (7, u'C'),
```
Comment (KentShikama) [inline aspc/courses/templatetags/stars.py]: @ZiqiXiong Actually one second. Can you import POSSIBLE_GRADES from the other file, so we aren't duplicating this?

**Comment by ZiqiXiong** [comment]
@KentShikama The grade is optional with default to be None, so users have the freedom to decide. Besides, the old course review did store grade, so it isn't a new change of dynamic.

File: aspc/courses/templatetags/stars.py (hunk 2 above)
Comment (ZiqiXiong) [inline aspc/courses/templatetags/stars.py]: Done

**Pull request merged**